README.md

```diff
@@ -92,6 +92,15 @@ Then run the binary in development mode with:
 $ event-gateway -dev
 ```
 
+### Kubernetes
+
+We've added `helm` charts to the repo for a quick deploy to an existing cluster. Please note, the default
```

Comment:
Maybe instead of writing "we've added" we could change it to something like "There are"